src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx

```diff
@@ -218,4 +219,28 @@ describe('526 helpers', () => {
       });
     });
   });
+
```

Comment:
Nothing wrong with these, but I wonder if they duplicate tests for the underlying `get` util.

Reply:
Since we don't test the `depends` in any other way, I wanted to make sure we'd catch it if we changed the form data structure without updating the path used in the `depends` callback.